webui/react/src/e2e/tests/experimentList.spec.ts

```diff
@@ -52,6 +52,7 @@ test.describe('Experiement List', () => {
     const row = await projectDetailsPage.f_experiemntList.dataGrid.getRowByColumnValue('ID', '1');
     await row.clickColumn('Select');
     expect(await row.isSelected()).toBeTruthy();
+    await expect((await row.getCellByColumnName('Checkpoints')).pwLocator).toHaveText('0');
```

Comment:
this would fail if the table isn't scrolled to the row. in the DOM, the column wouldn't exist.